code/controllers/CMSPageAddController.php

```diff
@@ -21,7 +21,7 @@ public function AddForm() {
 		$pageTypes = array();
 		foreach($this->PageTypes() as $type) {
 			$html = sprintf('<span class="page-icon class-%s"></span><strong class="title">%s</strong><span class="description">%s</span>',
-				$type->getField('Title'),
+				$type->getField('ClassName'),
```

Comment:
What's the purpose of this change? Doesn't seem related to the rest of the patch?